packages/react-native/package.json

```diff
@@ -11,7 +11,7 @@
     "build:cjs": "tsc --project tsconfig.cjs.json && cp -a src/assets/icons lib/assets",
     "build:esm": "tsc --project tsconfig.esm.json && cp -a src/assets/icons dist/assets",
     "clean": "rimraf dist",
-    "dev": "yarn build:esm --watch",
+    "dev": "yarn build && tsc --project tsconfig.esm.json --watch",
```

Comment:
@calebpollman this command was failing b/c '--watch was not a directory' b/c the `cp` command came after the `tsc` command.

Reply:
Nice catch!